app/models/friend.rb

```diff
@@ -5,29 +5,26 @@
 require "smartystreets_ruby_sdk/us_street/match_type"
 # require 'dry/types'
 # require_relative '../lib/tasks/decimal_types'
+# require 'colorize' # Extends String class
+# require 'colorized_string' # add ColorizedString class
```

Comment:
cruft.

I get that you want to use these, but they should be introduced in the branch that introduces that functionality.


PRs (pull requests) should be constrained to single issues to keep code changes clean and understandable.

Reply:
I am not going to back track at this point. I will store that bit of knowledge away for future api's like the rewrite of this one.